src/EnergyPlus/SQLiteProcedures.hh

```diff
@@ -104,7 +104,6 @@ protected:
 
     bool m_writeOutputToSQLite;
     std::shared_ptr<std::ostream> m_errorStream;
-    sqlite3 *m_connection;
     std::shared_ptr<sqlite3> m_db;
 };
```

Comment:
Note that `m_db` is a shared pointer to `m_connection`, created in the SQLiteProcedures constructor. There is no need to store `m_connection` for direct access.